subprojects/docs/src/docs/release/notes.md

```diff
@@ -46,6 +46,24 @@ Garbage collection time goes from [2.6 seconds](https://scans.gradle.com/s/3bg67
 
 While the impact on your build may vary, most builds can expect a noticeably shorter feedback loop when editing Kotlin DSL build logic thanks to this improvement.
 
+### Improved cache hits with normalized runtime classpaths
```

Comment:
This is actually copied from the feature description for the meta-inf normalization in 6.6.  I think it was added during release notes polishing, but I agree it sounds misleading.  I've changed it slightly to make it sound more like it's an improvement to normalization rather than that normalization is the feature.